### PR TITLE
Release notes for ES6

### PR DESCRIPTION
**Pull request by mburke5678**

### CODE REVIEW

**Comment by mburke5678** [comment]
@sichvoge I have started a release note for the ES6 changes and for retention policy. If you could help with the messaging here, I would greatly appreciate it.

File: release_notes/ocp-4-5-release-notes.adoc [hunk 1]
```diff
@@ -126,6 +126,29 @@ cluster installation by sharing the node's pull secret credentials with the
...
 
+[id="ocp-4-5-cluster-logging"]
+=== Cluster logging
+
+[id="ocp-4-5-elasticsearch-6"]
+==== Elasticsearch version upgrade
+
+Cluster logging in {product-title} {product-version} now uses Elasticseach 6.8.1 as the default log store. 
+
+The new Elasticsearch version introduces a new Elasticsearch data model that improves overall performance and scalability. With the new data model, data is no longer indexed by type, infrastructure, application and project. Data is only indexed by type:
```
Comment (jcantrill) [inline release_notes/ocp-4-5-release-notes.adoc:137]: ...no longer indexed by type(e.g. infrastructure, application) and project.

File: release_notes/ocp-4-5-release-notes.adoc [hunk 2]
```diff
@@ -126,6 +126,29 @@ cluster installation by sharing the node's pull secret credentials with the
...
+
+* The application logs that were previously in the *project-* indicies in {product-title} 4.4 are now in a single *app-* index.
```
Comment (jcantrill) [inline release_notes/ocp-4-5-release-notes.adoc:139]: are now in a set of indices prefixed with *app-*.

File: release_notes/ocp-4-5-release-notes.adoc [hunk 3]
```diff
@@ -126,6 +126,29 @@ cluster installation by sharing the node's pull secret credentials with the
...
+* The infrastructure logs that were previously in the *.opertions-* indices are now in the *infra-* indices. 
+* The audit logs are stored in the *audit-* indices.
+
+Because of the new data model, you will need to update your Kibana index patterns and visualizations to match the new indices.
+
+Elasticserach 6.x also introduces an enhanced security feature, Open Distro for Elasticsearch. Open Distro for Elasticsearch provides a comprehensive set of advanced security features designed to keep your data secure.
```
Comment (jcantrill) [inline release_notes/ocp-4-5-release-notes.adoc:145]: introduces an alternate security plugin

File: release_notes/ocp-4-5-release-notes.adoc [hunk 4]
```diff
@@ -126,6 +126,29 @@ cluster installation by sharing the node's pull secret credentials with the
...
+
+[id="ocp-4-5-log-retention-policy"]
+==== New Elasticsearch log retention feature
+
+The new index management feature allows you to configure when Elasticsearch performs data rollovers and how log Elasticsearch retains the rolled-over data. The index management feature replaces Curator. In {product-title} product-version}, Curator removes data that is in the Elasticsearch index formats prior to {product-title} 4.5 and will be removed in a later release.
```
Comment (jcantrill) [inline release_notes/ocp-4-5-release-notes.adoc:150]: The new index management relies on Elasticsearch rollover feature to maintain indices.  You can configure how log to retain data before it is removed from the cluster. The index management feature replaces ....

**Review by jcantrill — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-5-release-notes.adoc (hunk 3 above)
Comment (mburke5678) [reply release_notes/ocp-4-5-release-notes.adoc:145]: @jcantrill Thank you for the plugin wording. Was looking for that. 
It's not truly _alternate_, right? That sounds like I have a choice.
_introduces a new security plugin_?

File: release_notes/ocp-4-5-release-notes.adoc (hunk 4 above)
Comment (mburke5678) [reply release_notes/ocp-4-5-release-notes.adoc:150]: @jcantrill If I understand correctly, one of the rollover triggers is the `rollover.maxAge` setting in the ES CR. Am i correct that this value is derived using the `maxAge` that the user sets in the cluster logging CR retention policy? This is what I suggested _feature allows you to configure when Elasticsearch performs data rollovers_. 
If so, the wording could be better, _the feature allows you to influence one of the rollover triggers_ or something like that.

File: release_notes/ocp-4-5-release-notes.adoc [hunk 5]
```diff
@@ -297,6 +297,29 @@ availability.
...
+
+Cluster logging in {product-title} {product-version} now uses Elasticseach 6.8.1 as the default log store. 
+
+The new Elasticsearch version introduces a new Elasticsearch data model. With the new data model, data is no longer indexed by type (infrastructure and application) and project. Data is only indexed by type:
+
+* The application logs that were previously in the *project-* indicies in {product-title} 4.4  are in a set of indices prefixed with *app-*.
+* The infrastructure logs that were previously in the *.opertions-* indices are now in the *infra-* indices. 
+* The audit logs are stored in the *audit-* indices.
+
+Because of the new data model, you will need to update your Kibana index patterns and visualizations to match the new indices.
```
Comment (codyhoag) [inline release_notes/ocp-4-5-release-notes.adoc:314]: s/you will need to/you must

File: release_notes/ocp-4-5-release-notes.adoc [hunk 6]
```diff
@@ -297,6 +297,29 @@ availability.
...
+
+Elasticserach 6.x also includes a new security plug-in, Open Distro for Elasticsearch. Open Distro for Elasticsearch provides a comprehensive set of advanced security features designed to keep your data secure.
+
+[id="ocp-4-5-log-retention-policy"]
+==== New Elasticsearch log retention feature
+
+The new index management feature relies on the Elasticsearch rollover feature to maintain indices. You can configure how log to retain data before it is removed from the cluster. The index management feature replaces Curator. In {product-title} {product-version}, Curator removes data that is in the Elasticsearch index formats prior to {product-title} {product-version}, and will be removed in a later release.
```
Comment (codyhoag) [inline release_notes/ocp-4-5-release-notes.adoc:321]: s/log/long

File: release_notes/ocp-4-5-release-notes.adoc [hunk 7]
```diff
@@ -297,6 +297,29 @@ availability.
...
+=== Cluster logging
+
+[id="ocp-4-5-elasticsearch-6"]
+==== Elasticsearch version upgrade
+
+Cluster logging in {product-title} {product-version} now uses Elasticseach 6.8.1 as the default log store. 
+
+The new Elasticsearch version introduces a new Elasticsearch data model. With the new data model, data is no longer indexed by type (infrastructure and application) and project. Data is only indexed by type:
+
+* The application logs that were previously in the *project-* indicies in {product-title} 4.4  are in a set of indices prefixed with *app-*.
```
Comment (codyhoag) [inline release_notes/ocp-4-5-release-notes.adoc:310]: Remove extra space after `4.4`

**Review by codyhoag — COMMENTED** [review]
A few minor suggestions; otherwise, LGTM!

**Pull request merged**